arctic/chunkstore/chunkstore.py

```diff
@@ -65,6 +65,10 @@ def _ensure_index(self):
                                        (START, pymongo.ASCENDING),
                                        (SEGMENT, pymongo.ASCENDING)],
                                       unique=True, background=True)
+        self._mdata.create_index([(SYMBOL, pymongo.ASCENDING),
+                                  (START, pymongo.ASCENDING),
+                                  (END, pymongo.ASCENDING)],
```

Comment:
How come you have the `START` `END` on the metadata, doesn't the metadata apply to the whole symbol, rather than a range?